modules/sbo-creating-a-postgresql-database-instance-power-z.adoc

```diff
@@ -6,27 +6,16 @@ To create a PostgreSQL database instance, you must create a `Database` custom re
 
 .Procedure
 
-. Create the `Database` CR and the `my-postgresql` namespace by running the following command in shell:
+. Create the `Database` CR in the `my-petclinic` namespace by running the following command in shell:
 +
 [source,terminal]
 ----
-$ oc apply -f - << EOD
+$ oc apply -n my-petclinic -f - << EOD
 apiVersion: postgresql.dev4devs.com/v1alpha1
 kind: Database
 metadata:
   name: sampledatabase
-  namespace: my-postgresql
-  annotations:
-    host: sampledatabase
-    type: postgresql
-    port: "5432"
-    service.binding/database: 'path={.spec.databaseName}'
-    service.binding/port: 'path={.metadata.annotations.port}'
-    service.binding/password: 'path={.spec.databasePassword}'
-    service.binding/username: 'path={.spec.databaseUser}'
-    service.binding/type: 'path={.metadata.annotations.type}'
-    service.binding/host: 'path={.metadata.annotations.host}'
-spec:
+ spec:
```

Comment:
this should not change - that introduces a wrong indentation and the apply fails